locales/templates/app.pot

```diff
@@ -1202,3 +1251,159 @@ msgctxt "custom_tab_copy_url_action"
+#. previous session is cleared, and authentication is not required.
+msgctxt "biometric_auth_new_session"
+msgid "New Session"
+msgstr ""
+
+#. Description for the fingerprint authentication image. Used for
+#. accessibility purposes.
+msgctxt "biometric_auth_image_description"
+msgid "Fingerprint icon"
+msgstr ""
+
+#. Error text displayed when the user's fingerprint cannot be recognized upon
+#. re-entering the app.      This is displayed during the authentication flow
+#. as the app launches.
+msgctxt "biometric_auth_not_recognized_error"
+msgid "Fingerprint not recognized. Try again."
+msgstr ""
+
+#. Error text displayed when the user's fingerprint moved too quickly upon
+#. re-entering the app     This is displayed during the authentication flow as
+#. the app launches.
+msgctxt "biometric_auth_moved_too_quickly"
+msgid "Finger moved too fast. Try again."
+msgstr ""
+
+#. Title for the prompt to enable search suggestions
+msgctxt "enable_search_suggestion_title"
+msgid "Would you like to turn on search suggestions?"
+msgstr ""
+
+#. Description for the prompt to enable search suggestions          %1$s will
+#. be replaced by the app name (e.g. Firefox Focus)           %2$s will be
+#. replaced with the "Learn more" string with the name:
+#. enable_search_suggestion_subtitle_learnmore
+#, c-format
+msgctxt "enable_search_suggestion_subtitle"
+msgid ""
+"To get search suggestions, %1$s needs to send what you type in the "
+"address bar to the search engine you've chosen. %2$s."
+msgstr ""
+
+#. Learn more string to be placed inside: enable_search_suggestion_subtitle
+msgctxt "enable_search_suggestion_subtitle_learnmore"
+msgid "Learn more"
+msgstr ""
+
+#. Label used for the no button for the prompt to enable search suggestions
+msgctxt "enable_search_suggestions_no"
+msgid "No"
+msgstr ""
+
+#. Label used for the yes button for the prompt to enable search suggestions
+msgctxt "enable_search_suggestions_yes"
+msgid "Yes"
+msgstr ""
+
+#. Message displayed when using a search engine that doesn't provide
+#. suggestions
+msgctxt "no_suggestions_message"
+msgid "Some search engines cannot show suggestions."
+msgstr ""
+
+#. Button to dismiss the message displayed  when using a search engine that
+#. doesn't provide suggestions
+msgctxt "dismiss_no_suggestions_prompt_button"
+msgid "Dismiss"
+msgstr ""
+
+#. Tips displayed on the home view
+#. Tip displayed on home view explaining how to disable tracking protection
+#. The newline here must be kept as the second half of the string is clickable
+#. for the user.
+msgctxt "tip_disable_tracking_protection"
+msgid ""
+"Site behaving unexpectedly?\n"
+" Try turning off Tracking Protection"
+msgstr ""
+
+#. Tip displayed on home view explaining how to add sites to homescreen
+#. The newline here must be kept as the second half of the string is clickable
+#. for the user.
+msgctxt "tip_add_to_homescreen"
+msgid ""
+"Get one-tap access to sites you use most\n"
+" Menu &gt; Add to Home screen"
```

Comment:
I think the tag should be converted? Not sure. We use that in Firefox / Fennec strings, but haven't seen it in other products

Reply:
Hmm the string in the source code is `>`. Not sure why it's getting converted here
```
<string name="tip_add_to_homescreen">Get one-tap access to sites you use most\n
        Menu > Add to Home screen</string>
```